<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>edit_Cycles</class>
 <widget class="QDialog" name="edit_Cycles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>444</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Edit cycles settings</string>
  </property>
  <widget class="QDialogButtonBox" name="dialogButtonBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>380</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="tb_Setpoint2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>190</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="l_Cycle1_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>76</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cycle 1:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tb_Cycle2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>190</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="tb_Setpoint1">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>110</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="l_Setpoint_Cycle_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>190</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Setpoint 2:</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_Cycle2_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cycle 2:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tb_Cycle1">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="l_Setpoint_Cycle_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>110</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Setpoint 1:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_Cycle_Mode_Toggle">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>50</y>
     <width>236</width>
     <height>22</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Cycles mode enabled</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dialogButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>edit_Cycles</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialogButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>edit_Cycles</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
